### DISCUSSION TITLE
NBS List

### DISCUSSION THREAD

**Discussion by hrishiballal** [nbsapi/nbsapi#6] [Ideas]
I think that the end points at the moment are [great](https://github.com/urschrei/nbsapi/blob/main/src/nbsapi/api.py#L105), what I am wondering is if we should enable querying by category e.g. flood / heat rather than the type of NBS.

**Comment by urschrei** [comment]
The `/trees` and `/bioswales` endpoints are really just examples for now – https://nbs-api.fly.dev/redoc#operation/get_solutions_solutions_get now requires a category enum query param
